conf/app.routes

```diff
@@ -30,4 +30,5 @@ GET         /requested/:fileRole                         controllers.HistoricSta
 GET         /requested/duty-deferment/:linkId            controllers.HistoricStatementsController.historicStatementsDutyDeferment(linkId: String)
 
 GET         /logout-survey                               controllers.LogoutController.logout
-GET         /logout                                      controllers.LogoutController.logoutNoSurvey
\ No newline at end of file
+GET         /logout                                      controllers.LogoutController.logoutNoSurvey
+GET         /verify-your-email                       controllers.EmailController.showUnverified()
```

Comment:
Could you please align the formatting for controllers.EmailController.showUnverified()?